projects/1.16/assets/neoelfeos-medieval-pub-decoration/nefdecomod/lang/zh_cn.json

```
@@ -0,0 +1,377 @@
+  "block.nefdecomod.mushroom_stew_4": "炖蘑菇",
+  "block.nefdecomod.mushroom_stew_6": "炖蘑菇",
+  "block.nefdecomod.oak_cava": "酒架",
+  "block.nefdecomod.mushroom_stew_3": "炖蘑菇",
+  "advancements.battlefield_adv.title": "无人\u0027s之地",
+  "block.nefdecomod.mushroom_stew_2": "炖蘑菇",
+  "item.nefdecomod.reinforced_chainmail_boots": "加固锁链靴子",
+  "block.nefdecomod.veg_stew": "炖蔬菜",
+  "block.nefdecomod.clay_pot": "粘土锅",
+  "block.nefdecomod.logs_6x_3": "木材堆",
+  "block.nefdecomod.bucket": "水桶",
+  "item.nefdecomod.ravager_helmet_leggings": "劫掠者头盔护腿",
+  "block.nefdecomod.ropes": "绳网",
+  "block.nefdecomod.branch_3_block": "分叉树枝",
+  "entity.nefdecomod.chair": "椅子",
+  "block.nefdecomod.wardrobe_5": "衣柜",
+  "block.nefdecomod.wardrobe_4": "衣柜",
+  "block.nefdecomod.wardrobe_6": "衣柜",
+  "block.nefdecomod.wardrobe_1": "衣柜",
+  "block.nefdecomod.wardrobe_3": "衣柜",
+  "block.nefdecomod.wardrobe_2": "衣柜",
+  "advancements.medieval_bar.descr": "我听说附近有酒馆……",
+  "block.nefdecomod.world_map": "地球仪",
+  "block.nefdecomod.carrots": "胡萝卜堆",
+  "block.nefdecomod.target": "箭靶",
+  "block.nefdecomod.mushroom_stew_crude": "生炖蘑菇",
+  "item.nefdecomod.baked_nopal": "烤胭脂仙人掌",
+  "block.nefdecomod.clay_mess": "粘土废料",
+  "itemGroup.tabmedievalpub": "中世纪装饰丨酒馆",
+  "block.nefdecomod.log_1_d": "木材",
+  "block.nefdecomod.susp_stew_raw": "生蔬菜炖蘑菇",
+  "block.nefdecomod.cage_linked": "笼子锁链",
+  "block.nefdecomod.taburete": "凳子",
+  "block.nefdecomod.few_stones": "稀疏石块堆",
+  "block.nefdecomod.branch_2_block": "分叉树枝",
+  "block.nefdecomod.bread": "面包堆",
+  "block.nefdecomod.shackles": "手铐脚镣",
+  "block.nefdecomod.fruit_box_cantaloupe": "哈密瓜水果盒",
+  "item.nefdecomod.veg_stew_food": "炖蔬菜",
+  "block.nefdecomod.fruit_box_potatoes": "土豆水果盒",
+  "block.nefdecomod.salt_block": "盐块",
+  "block.nefdecomod.coffin_dead": "棺材",
+  "block.nefdecomod.fruit_box_pumpkin": "木盒（南瓜）",
+  "item.nefdecomod.radish": "小萝卜",
+  "block.nefdecomod.sign_0": "招牌",
+  "itemGroup.tabplushies": "中世纪装饰丨玩偶",
+  "item.nefdecomod.hand_saw": "手锯",
+  "block.nefdecomod.apples": "苹果堆",
+  "block.nefdecomod.chela_plushie": "Chela 玩偶",
+  "block.nefdecomod.desk": "桌子（空）",
+  "block.nefdecomod.desk_full_b": "桌子（满）",
+  "block.nefdecomod.desk_full_a": "桌子（满）",
+  "block.nefdecomod.logs_3slab": "木材堆",
+  "block.nefdecomod.desk_full_c": "桌子（满）",
+  "block.nefdecomod.logs_2rest": "木材",
+  "block.nefdecomod.log_6rest": "木材",
+  "block.nefdecomod.oak_barrel": "水桶",
+  "block.nefdecomod.melon_slices": "西瓜堆",
+  "block.nefdecomod.adobe_double_wall": "粘土砖块",
+  "item.nefdecomod.book_2_item": "智之魔导书",
+  "block.nefdecomod.nef_dead": "Nef 的尸体",
+  "item.nefdecomod.reinforced_leather_leggings": "强化皮护腿",
+  "block.nefdecomod.mug_fallen": "倒下的马克杯",
+  "block.nefdecomod.jungle_table": "桌子",
+  "block.nefdecomod.birch_cava_b": "酒架",
+  "block.nefdecomod.veg_stew_4": "炖蔬菜",
+  "block.nefdecomod.veg_stew_5": "炖蔬菜",
+  "block.nefdecomod.veg_stew_2": "炖蔬菜",
+  "advancements.battlefield_adv.descr": "你进入了一个冲突区域",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "advancements.battlefield_adv.descr": "你进入了一片冲突区域",
```